buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -290,20 +290,20 @@ class ClusterFormationTasks {
         Copy copyConfig = project.tasks.create(name: name, type: Copy, dependsOn: setup)
         File configDir = new File(node.homeDir, 'config')
         copyConfig.into(configDir) // copy must always have a general dest dir, even though we don't use it
-        for (Map.Entry<String,Object> extraConfigFile : node.config.extraConfigFiles.entrySet()) {
```

Comment:
In the other groovy files for buildSrc, I've been trying to use as little groovy stuff as possible. Can we keep the simple for loop here? It seems the changes to this file are just cosmetic.